libglusterfs/src/statedump.c

```diff
@@ -915,18 +915,15 @@ gf_proc_dump_info(int signum, glusterfs_ctx_t *ctx)
     // continue even though gettimeofday() has failed
     ret = gettimeofday(&tv, NULL);
     if (0 == ret) {
-        gf_time_fmt_tv_FT(timestr, sizeof timestr, &tv, ctx);
+        gf_time_fmt_tv_FT(timestr, sizeof timestr, &tv, &ctx->log);
         len = snprintf(sign_string, sizeof(sign_string),
                        "DUMP-START-TIME: %s\n", timestr);
 
         // swallow the errors of write for start and end marker
         (void)sys_write(gf_dump_fd, sign_string, len);
+        memset(timestr, 0, sizeof(timestr));
```

Comment:
Why is this needed ?

Reply:
I don't think it's needed - it was there. Just as I don't think we need to initialize timestr everywhere to a char array full with NULLs.... Should I remove it?